### PR TITLE
Update the menu for the Dialogs

### PR DESCRIPTION
**Pull request by singa2000**
[Issue] N/A
[Problem]There was existing outdate examples
[Solution]Remove outdate examples and update it followed by UX guide

Signed-off-by: cookie <cookie@samsung.com>

### CODE REVIEW

**Comment by singa2000** [comment]
Dialog  Picker sample is empty because it's being implemented.
If implementation is done, it'll make another PR.

File: examples/mobile/UIComponents/components/popup/index.html [hunk 1]
```diff
@@ -13,44 +13,19 @@
 	<div class="ui-page" id="popup-demo-page">
 		<div class="ui-header" data-position="fixed">
 			<h1>
-				Popup
+				Dialogs
```
Comment (HunseopJeong) [inline examples/mobile/UIComponents/components/popup/index.html:16]: Folder name is the 'popup' but title and ux guide are 'Dialogs'.
In wearable, it seems to be used as popup :(
How about change the folder name also?

Comment (singa2000) [reply examples/mobile/UIComponents/components/popup/index.html:16]: Sure. I'll change the folder name popup-> Dialog according to UX guide.

File: examples/mobile/UIComponents/components/dialogs/picker.html [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+						Date and time picker (12h)
+					</a>
+				</li>
+				<li class="ui-li-anchor">
+					<a data-inline="true">
+						Date and time picker (24h)
+					</a>
+				</li>
+			</ul>
+
```
Comment (DohyungLim) [inline examples/mobile/UIComponents/components/dialogs/picker.html]: remove 1 line.

File: examples/mobile/UIComponents/components/dialogs/picker.html [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+		</div>
+	</div>
+</body>
+
+</html>
```
Comment (DohyungLim) [inline examples/mobile/UIComponents/components/dialogs/picker.html]: check to end-line.

File: examples/mobile/UIComponents/components/dialogs/popup.html [hunk 4]
```diff
@@ -13,188 +13,154 @@
...
+					Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
+					<br />
+					<input checked="" type="checkbox" />
+					Do not show again
 				</div>
-			</div>
-			<div class="ui-popup" id="footer_popup">
-				<div class="ui-popup-header">
-					Popup Header
+				<div class="ui-popup-footer">
```
Comment (DohyungLim) [inline examples/mobile/UIComponents/components/dialogs/popup.html]: Is indent applied well?

**Review by DohyungLim — COMMENTED** [review]
I already have a sample of Dialogs. 
but, thank you for organizing the page according to the spec docs.
please check the comments. 
and if there are no problems, merge so I can insert the code.

File: examples/mobile/UIComponents/components/dialogs/index.html [hunk 5]
```diff
@@ -13,44 +13,19 @@
...
-				Popup
+				Dialogs
 			</h1>
 		</div>
 		<div class="ui-content">
 			<ul class="ui-listview ui-content-area">
 				<li class="ui-li-anchor">
 					<a href="popup.html">
-						Popup
+						General Dialogs
```
Comment (DohyungLim) [inline examples/mobile/UIComponents/components/dialogs/index.html:23]: Check whether it is General Dialog / Dialog.

**Review by DohyungLim — COMMENTED** [review]
1 more comment.

File: examples/mobile/UIComponents/components/dialogs/index.html (hunk 5 above)
Comment (singa2000) [reply examples/mobile/UIComponents/components/dialogs/index.html:23]: I make a "General Dialogs" for distinguish with Picker Dialog.
If it confuses you, I'll change "Dialog".

**Review by HunseopJeong — APPROVED** [review]
LGTM. Need to check @DohyungLim's comments.

File: examples/mobile/UIComponents/components/dialogs/dialog.html [hunk 6]
```diff
@@ -13,188 +13,154 @@
...
+				<li class="ui-li-anchor">
+					<a data-position-to="window" data-rel="popup" href="#2btn_notitle_popup">
+						2btn Notitle Dialog
 					</a>
 				</li>
-				<li class="ui-group-index">
-					List
+				<li class="ui-li-anchor">
+					<a data-position-to="window" data-rel="popup" href="#nobtn_title_popup">
+						Nobtn Title Dialog
```
Comment (DohyungLim) [inline examples/mobile/UIComponents/components/dialogs/dialog.html]: I think that replace "No" to 0 or nothing.

File: examples/mobile/UIComponents/components/dialogs/dialog.html [hunk 7]
```diff
@@ -13,188 +13,154 @@
...
+				<li class="ui-li-anchor">
+					<a data-position-to="window" data-rel="popup" href="#3Stackbtn_popup">
+						3StackBtn Title Dialog
 					</a>
 				</li>
-				<li class="ui-group-index">
-					Content
+				<li class="ui-li-anchor">
+					<a data-position-to="window" data-rel="popup" href="#nobtn_selectlist_popup">
+						Nobtn Title Select List Dialog
```
Comment (DohyungLim) [inline examples/mobile/UIComponents/components/dialogs/dialog.html]: ditto.

File: examples/mobile/UIComponents/components/dialogs/dialog.html [hunk 8]
```diff
@@ -13,188 +13,154 @@
...
-					<a data-position-to="window" data-rel="popup" href="#1btn_popup">
-						1btn popup
-					</a>
-				</li>
-				<li>
+			<ul class="ui-listview ui-content-area">
+				<li class="ui-li-anchor">
 					<a data-position-to="window" data-rel="popup" href="#2btn_popup">
-						2btn popup
+						2btn Title Dialog
```
Comment (DohyungLim) [inline examples/mobile/UIComponents/components/dialogs/dialog.html]: Separate numbers before every button. It's easy to recognize this.
Apply to all code.

File: examples/mobile/UIComponents/components/dialogs/picker.html (hunk 3 above)
Comment (DohyungLim) [reply examples/mobile/UIComponents/components/dialogs/picker.html]: check point.

File: examples/mobile/UIComponents/components/dialogs/index.html (hunk 5 above)
Comment (DohyungLim) [reply examples/mobile/UIComponents/components/dialogs/index.html:23]: ok. using General Dialogs. agree.

**Review by DohyungLim — COMMENTED** [review]
5 comment check.

**Review by DohyungLim — APPROVED** [review]
LGTM.

**Pull request merged**